mongodbatlas/resource_mongodbatlas_third_party_integration.go

```diff
@@ -21,11 +21,6 @@ var integrationTypes = []string{
 	"PROMETHEUS",
 }
 
-var deprecatedIntegrationTypes = []string{
```

Comment:
Do you have to update the `mongodbatlas/resource_mongodbatlas_third_party_integration_test.go` ?

Reply:
this piece of logic with the deprecated integration types was not being tested so I just removed it. Let me know if you feel like this need more tests

Reply:
Could you check if we have an acceptance test scenario for all the integrations and if not, add the scenario? No need to address this comment here but you can open a follow-up PR. Thanks!